src/clientsettingsdlg.cpp

```diff
@@ -1009,29 +1010,19 @@ void CClientSettingsDlg::OnEnableOPUS64StateChanged ( int value )
 
 void CClientSettingsDlg::OnFeedbackDetectionChanged ( int value ) { pSettings->bEnableFeedbackDetection = value == Qt::Checked; }
 
-void CClientSettingsDlg::OnCustomDirectoriesEditingFinished()
+void CClientSettingsDlg::OnCustomDirectoriesChanged ( bool bDelete )
 {
```

Comment:
`cbxCustomDirectories->currentData().isValid()` looks common to both legs of the `if` -- so the method could just return (doing nothing) if it's false.